metadata-service/servlet/src/main/java/com/datahub/gms/servlet/Config.java

```diff
@@ -31,6 +32,7 @@ public class Config extends HttpServlet {
     put("retention", "true");
     put("statefulIngestionCapable", true);
     put("patchCapable", true);
+    put("timeZone", ZoneId.systemDefault());
```

Comment:
what's the use in this? wouldn't this be the timezone that the server is in and not necessarily the user?